app/controllers/course/stage/instructions.ts

```diff
@@ -54,13 +54,17 @@ export default class CourseStageInstructionsController extends Controller {
   }
 
   get shouldShowLanguageGuide() {
-    return !this.model.courseStage.isFirst && !!this.languageGuide && this.featureFlags.canSeeLanguageGuidesForStage2;
+    return !this.model.courseStage.isFirst && !!this.languageGuide;
   }
 
   get shouldShowPrerequisites() {
     return !!this.prerequisiteInstructionsMarkdown;
   }
 
+  get shouldShowSolution() {
```

Comment:
We're in "instructions.hbs" here, so this could apply to any stage. Since our feature flag is only for stage 2, maybe we could do this: 

```suggestion
  get shouldShowStage2Solution() {
```